src/app/main-overview/components/MainOverview.vue

```diff
@@ -1,36 +1,44 @@
 <template>
   <DataSource
-    v-slot="{ data: meshInsightsData, isLoading: meshInsightsIsLoading, error: meshInsightsError }: MeshInsightCollectionSource"
-    src="/all-mesh-insights"
+    v-slot="{ data, isLoading, error }: GlobalInsightsSource"
+    src="/global-insights"
   >
```

Comment:
Did we say that in the future this will become a single call to `/global-insights` ?

Reply:
With what we show on this page, we definitely need all three sources. Global insights is the data source for the top card containing the online figures. Zone insights and mesh insights are for the table respectively. Note how for both, we only request at most 10 (i.e. the `size` query parameter is `10`).

Reply:
Yeah I agree. What I was hoping is that all 3 Datasource could be declared close to their usage and then all 3 requests would be done at once.

Reply:
They will be done in parallel still. Nesting DataSources like that doesn’t mean they are processed in series. That would only be the case if we enforce that via `v-if` conditions on them.

Reply:
Is that true for the rendering as well? And do you not think that would improve readability?

Reply:
> Is that true for the rendering as well?

Pretty much, yes.

> And do you not think that would improve readability?

To me it doesn’t really help with readability moving the DataSources closer to where their data is used, no. I find it rather a distraction.

Regardless, I’ve made a change that does so in https://github.com/kumahq/kuma-gui/pull/1315/commits/24df7dd2f0e1811858ca80b6e7aa1c05fc19042a.